Eatery/Views/Eateries/HomescreenViews/EateryCollectionViewCell.swift

```diff
@@ -263,6 +284,57 @@ class EateryCollectionViewCell: UICollectionViewCell {
 
         timeLabel.textColor = .lightGray
         distanceLabel.textColor = .lightGray
+        let favoriteFoods = eatery.getFavorites().sorted()
+        if favoriteFoods.count > 0 {
+            let font = UIFont.systemFont(ofSize: 14)
+            let paragraphStyle = NSMutableParagraphStyle()
+            paragraphStyle.paragraphSpacing = 0.25 * font.lineHeight
+
+            var favoritesList = [NSMutableAttributedString]()
+
+            for food in favoriteFoods {
+                var name: NSMutableAttributedString = NSMutableAttributedString(
+                    string: " \(food.trim())  ",
+                    attributes: [
+                        .foregroundColor: UIColor.gray,
+                        .font: font,
+                        .paragraphStyle: paragraphStyle
+                    ]
+                )
+                if let image = UIImage.favoritedImage?.withRenderingMode(.automatic) {
+                    name = name.prependImage(image, yOffset: -1.5, scale: 0.5)
+                }
+                favoritesList.append(name)
+            }
+            let newLine = NSAttributedString(string: "\n")
+            let favoritesText = NSMutableAttributedString()
+            if favoritesList.map({ $0.size().height }).reduce(0, +) < frame.height {
```

Comment:
```suggestion
            if favoritesList.map { $0.size().height }.reduce(0, +) < frame.height {
```

You can delete the `()` around the `{}` when it's a trailing closure.